src/types/suspiciousInfo.ts

```diff
@@ -0,0 +1,15 @@
+import { ScamInfoType } from './account.types';
+
+export interface SuspiciousInfo {
```

Comment:
suspicous is too vague. Maybe replace with SuspiciousLinkType
then also rename file to suspiciousLink.ts
and below SuspiciousLinkPropsType